spec/models/claim/base_claim_spec.rb

```diff
@@ -534,6 +534,40 @@ class MockSteppableClaim < Claim::BaseClaim
       end
     end
   end
+
+  describe '#trial_length' do
+    let(:actual_trial_length) { 3 }
+    let(:retrial_actual_length) { 5 }
+    let(:claim) { MockSteppableClaim.new(actual_trial_length: actual_trial_length, retrial_actual_length: retrial_actual_length) }
+
+    subject(:trial_length) { claim.trial_length }
+
+    context 'when the claim requires re-trial dates' do
+      before do
+        expect(claim).to receive(:requires_retrial_dates?).and_return(true)
+      end
+
+      specify { is_expected.to eq(retrial_actual_length) }
```

Comment:
any particular reason for using `specify` rather than `it`. since the specs generally use it and `it { is_expected.to }` makes for clear english

Reply:
In regards to this, I'm happy to create a separate PR to address it, even though they're formally equivalent 👍 .